auto_joiner.py

```diff
@@ -448,11 +460,13 @@ def join_meeting(meeting):
             send_button = wait_until_found("#send-message-button", 5)
             send_button.click()
             print(f'Sent message {config["join_message"]}')
+            discord_notification("Sent message", {config["join_message"]})
```

Comment:
Remove the {} brackets or put f"..." around it

Reply:
I missed the f there. Ill do that